services/explorer/src/utils/TransactionRow.tsx

```diff
@@ -181,3 +188,49 @@ export const toNodeTableRow = ({
     blocksProduced: pbftCount.toLocaleString('en-US'),
   };
 };
+
+export const toHolderTableRow = ({
+  rank,
+  address,
+  balance,
+  totalSupply,
+  taraPrice,
+}: HoldersTableData): {
+  rank: number;
+  address: JSX.Element;
+  balance: string;
+  percentage: JSX.Element;
+  value: string;
+} => {
+  const addressLink = (
+    <HashLink linkType={HashLinkType.ADDRESSES} width='auto' hash={address} />
+  );
+  // We need to multiply by an additional 100 to get some precision back from bignumber
+  // Original formula should be balance * 100 / totalSupply
+  // But the improved one that returns for smaller holders too is: balance * 100 * 100 / totalSupply
+  const _balance = balance.mul(BigNumber.from(10000));
```

Comment:
You can't use `.mul()` on a string I guess.

Reply:
I double-checked the maths and it works out.

Reply:
The function param is a string